charts/redpanda/values.yaml

```diff
@@ -427,47 +424,39 @@ listeners:
       # If true, the truststore file for this listener will be included in the ConfigMap
       requireClientAuth: false
   # Kafka API listeners
-  # The kafka listener group cannot be disabled
   kafka:
-    port: 9093
-    # Listeners internal to kubernetes service network
+    # The port for internal client connections (10092 is default to avoid conflicts with the external default port 9092)
+    port: 10092
     tls:
       # enabled: true
       cert: default
       requireClientAuth: false
-    # External listeners
     external:
-    # to disable external kafka listeners when the global `external` is enabled,
-    # replace this with an empty list, ie: `external: []`
       default:
-        port: 9094
-        # If unset, it will default to the type in the `external` section.
-        type: NodePort
-        # External port
-        # This listener port will be used on each kubernetes node
-        nodePort: 31092
+        # enabled: true
+        # The port used for external client connections
+        port: 9092
+        # If undefined, `listeners.kafka.external.port` is used
+        advertisedPorts:
+        - 31092
   # HTTP API listeners (aka PandaProxy)
-  # PandaProxy is a kafka client that connects to an endpoint from listeners.kafka.endpoints
   http:
     enabled: true
-    port: 8082
+    port: 9082
+    # TODO is this for internal and external clients? why can't internal connect to kafka internal, and each external connect to matchin kafka external? then this can be removed
```

Comment:
No, it's only for the internal listener. If you go down to the `external` section, you'll see that `default` has its own port. This is a requirement because the port that's connected to determines which listener profile (and advertised listener set) is being used.

Reply:
This is a TODO line that I meant to remove beforehand, I'll remove the line.

Reply:
This TODO does have a purpose though, so I will create a related issue.

`kakfaEndpoint` exists to connect the http listener internally to the proper kafka listener based on name of the listener. But we could automate this by ensuring the internal http listener connects to the internal kafka listener, and then each external http listener connects to the kafka listener of the same name as itself (which by default, the one external listener created for both http and kafka is named `default`).

https://github.com/redpanda-data/helm-charts/issues/284